### PR TITLE
[SLING-8272] Check if Caconfig Node Exists in JCR

### PR DESCRIPTION
**Pull request by ompandeyy**
This PR is dependent on https://github.com/apache/sling-org-apache-sling-caconfig-api/pull/2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -119,7 +119,7 @@
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.caconfig.api</artifactId>
-            <version>1.1.0</version>
+            <version>1.1.1</version>
```
Comment (kwin) [inline pom.xml]: Isn’t this supposed to be the latest SNAPSHOT? The last release does not yet define „has“

File: src/main/java/org/apache/sling/caconfig/impl/ConfigurationBuilderImpl.java [hunk 2]
```diff
@@ -398,4 +398,23 @@ public T convert(Resource resource, Class<T> clazz, String configName, boolean i
         }
     }
 
+ // --- Config Node Existence Check Support ---
+    @Override
+    public <T> boolean has(Class<T> clazz) {
+        final String name = getConfigurationNameForAnnotationClass(clazz);
+        if (log.isDebugEnabled()) {
+            log.debug("Get configuration for context path {}, name '{}', class {}", contentResource.getPath(), name, clazz.getName());
```
Comment (kwin) [inline src/main/java/org/apache/sling/caconfig/impl/ConfigurationBuilderImpl.java]: Rather „check configuration“

File: pom.xml (hunk 1 above)
Comment (ompandeyy) [reply pom.xml]: Changed this with the latest SNAPSHOT

File: src/main/java/org/apache/sling/caconfig/impl/ConfigurationBuilderImpl.java (hunk 2 above)
Comment (ompandeyy) [reply src/main/java/org/apache/sling/caconfig/impl/ConfigurationBuilderImpl.java]: Adjusted this.

**Pull request merged**